morpheus/_lib/include/morpheus/messages/memory/inference_memory_fil.hpp

```diff
@@ -34,7 +34,7 @@ namespace morpheus {
  * @{
  * @file
  */
-
+#pragma GCC visibility push(default)
```

Comment:
You now have 2 pushes and only one pop.

You can also use the macro `MORPHEUS_EXPORT` to declare it visible. Like `class MORPHEUS_EXPORT LLMContext`. I havent had good success with the GCC visibility push/pop and sometimes prefer the macro.